funfact/lang/_ast.py

```diff
@@ -68,6 +70,15 @@ def pow(base: _ASNode, exponent: _ASNode):
     def neg(x: _ASNode):
         '''elementwise negation'''
 
+    @primitive(precedence=5)
+    def kron(lhs: _ASNode, rhs: _ASNode):
+        '''indexless Kronecker product'''
```

Comment:
What is the syntax for an indexless Kronecker product? I don't see this explained in the PR.

Reply:
This is fully implemented with the `&` operator in a later PR #124. Honestly, I'm not completely satisfied with the use of `&`, because it has a very low operator precedence which is not consistent with the 'product' nature of the operation. I am not sure if there will be better options, though.